docs/command-completion.md

```diff
@@ -1,29 +1,15 @@
-Command Completion
-==================
+# Command Completion
 
-Shell command completion is provided by the script at 
-[/misc/completion/ipfs-completion.bash](../misc/completion/ipfs-completion.bash).
+Shell command completions can be generated by running one of the `ipfs commands completions`
+sub-commands.
 
+The simplest way to see it working is write the completions
+to a file and then source it:
 
-Installation
-------------
-The simplest way to see it working is to run 
-`source misc/completion/ipfs-completion.bash` straight from your shell. This
-is only temporary and to fully enable it, you'll have to follow one of the steps
-below.
-
-### Bash on Linux
-For bash, completion can be enabled in a couple of ways. One is to copy the 
-completion script to the directory `~/.ipfs/` and then in the file 
-`~/.bash_completion` add
 ```bash
-source ~/.ipfs/ipfs-completion.bash
+> ipfs commands completion bash > ipfs-completion.bash
+> source ./ipfs-completion.bash
```

Comment:
You could be more adventurous and suggest:
`eval "$( ipfs commands completion bash )"`
This way on go-ipfs upgrades the completion will magically update

Reply:
Hm... Yeah, that's a good idea! Shouldn't be _too_ slow.

Reply:
```
time cmd/ipfs/ipfs commands completion bash > /dev/null

real	0m0.072s
...
```

Reply:
Yeah, not bad. And if anyone actually cares, they can just write it to a file.